package.json

```diff
@@ -34,7 +34,7 @@
     "juggler-v3": "file:./deps/juggler-v3",
     "juggler-v4": "file:./deps/juggler-v4",
     "lodash": "^4.17.4",
-    "loopback-datasource-juggler": "^3.0.0 || ^4.0.0",
+    "loopback-datasource-juggler": "^3.0.0 || ^4.18.1",
```

Comment:
Won't it automatically use `^4.18.1` because of the `^`?

Reply:
The CI wasn't passing so I was trying everything I could do 😂